wrappers/LPI_wrapper.go

```diff
@@ -17,6 +17,7 @@ var lpiCodeToProtocol = map[uint32]godpi.Protocol{
 	8:   godpi.Ssh,
 	23:  godpi.Rpc,
 	1:   godpi.Smtp,
+	92:  godpi.Smtp,
```

Comment:
1 is normal SMTP, 92 maps to LPI_PROTO_INVALID_SMTP which is SMTP but for "SMTP flows that do not conform to the ​SMTP specifications correctly, usually due to SMTP commands or reply codes being sent out of order". LPI would detect some packets as that (possibly because I send it each packet as a flow), so I thought I'd include it.